brave/brave-core
# fix(wallet): Remove Bridge To Aurora Button

## Description 

Remove the `Bridge to Aurora` button since it is no longer needed.


Resolves <https://github.com/brave/brave-browser/issues/42943>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and navigate to the `Portfolio` screen.
2. View details for `Ethereum`.
3. You should no longer see a `Bridge to Aurora` button.

Before:

![Image](https://github.com/user-attachments/assets/dd111b8e-320b-491f-a06d-58ab4df37e0e)

After:

![Image](https://github.com/user-attachments/assets/68ab5726-913c-4c44-8ca6-3c0fd265e036)
